### DISCUSSION TITLE
Just a minor suggestion to change the syntax of the access modifier file to local

### DISCUSSION THREAD

**Discussion by iam3yal** [dotnet/csharplang#6012] [Language Ideas]
I think that the [upcoming access modifier file](https://github.com/dotnet/csharplang/issues/5529) sounds odd when contrasted to other access modifiers we have in the language and `local` sounds nicer in my opinion maybe because existing access modifiers are all both nouns and adjectives whereas `file` differs in that sense so `local` similar to other modifiers sounds better than `file` when it stand on its own as well as mix better with existing access modifiers should you want to do it in the future.

**Comment by jnm2** [comment]
I'd be afraid that `local` is less clear if it was to appear on a nested type. Or if types are allowed in the future to be declared inside method bodies next to local functions.
